app/scripts/modules/core/src/serverGroup/details/RunningTasks.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+
+
+import { Application } from 'core/application/application.model';
+import { IServerGroup } from 'core/domain';
+import { CollapsibleSection, robotToHuman } from 'core/presentation';
+import { displayableTasks } from 'core/task/displayableTasks.filter';
+import { StatusGlyph } from 'core/task/StatusGlyph';
+import { duration } from 'core/utils/timeFormatters';
+import { PlatformHealthOverrideMessage } from 'core/task/PlatformHealthOverrideMessage';
+
+export interface IRunningTasksProps {
+  application: Application;
+  serverGroup: IServerGroup;
+}
+
+export interface IRunningTasksState {
+}
+
+export class RunningTasks extends React.Component<IRunningTasksProps, IRunningTasksState> {
+  constructor(props: IRunningTasksProps) {
+    super(props);
+  }
+
+  public render() {
+    const { application, serverGroup } = this.props;
+
+    const tasks = serverGroup.runningTasks.sort((a, b) => a.startTime - b.startTime).map((task) => (
```

Comment:
Personal preference: I would rather have these `JSX.Elements` as named (SFC) components, then do the mapping down below